src/ngraph/pass/algebraic_simplification.cpp

```diff
@@ -534,7 +535,7 @@ static bool simplify_gather_shapeof(shared_ptr<Node> node)
             concat_inputs.push_back(gather);
         }
         auto shapeof_indices =
-            make_shared<opset3::ShapeOf>(gather->input_value(1).get_node_shared_ptr());
+            make_shared<opset3::ShapeOf>(gather->input_value(1).get_node_shared_ptr()->output(0));
```

Comment:
Again, should just be `input_valie(1)`.

Reply:
we need the first output of the node , this node is the first input of gather. Wasn't sure input_value(1) represents this, will take a look.